tests/bdp_spark.erl

```diff
@@ -0,0 +1,138 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+%% @doc A module to test Spark fail/recovery under BDP service manager.
+
+-module(bdp_spark).
+-behavior(riak_test).
+-export([confirm/0]).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-define(SERVICE_1, "spark-service-one").
+-define(SERVICE_2, "spark-service-two").
+-define(SPARK_MASTER_TYPE, "spark-master").
+-define(SPARK_IDENT_STRING, "bdp_spark_test").
```

Comment:
The ident string seemed to be ignored (or had no effect since my machine has no "bdp_spark_test" user in the spark environment, so it ran as my user. This hard-coded assumption caused test failures.